### PR TITLE
Add support for IAST SQL injection system tests

### PR DESCRIPTION
**Pull request by manuel-alvarez-alvarez**
## Description

Add system tests for IAST SQL injection detection

### CODE REVIEW

File: tests/appsec/iast/test_iast.py [hunk 1]
```diff
@@ -56,3 +67,19 @@ def test_insecure_md5_hash(self):
         r = self.weblog_get("/iast/insecure_hashing/test_md5_algorithm")
 
         interfaces.library.expect_iast_vulnerabilities(r, vulnerability_type="WEAK_HASH", evidence="md5")
+
+    def test_secure_sql(self):
```
Comment (robertomonteromiguel) [inline tests/appsec/iast/test_iast.py:71]: @missing_feature(library="nodejs", reason="Need to implement nodejs endpoint")

File: utils/build/docker/java/spring-boot/pom.xml [hunk 2]
```diff
@@ -89,7 +93,15 @@
 		    <artifactId>bcprov-jdk15on</artifactId>
 		    <version>1.70</version>
 		</dependency>
-
+        <dependency>
```
Comment (robertomonteromiguel) [inline utils/build/docker/java/spring-boot/pom.xml:96]: do we really need flyway?

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]

File: tests/appsec/iast/test_iast.py (hunk 1 above)
Comment (manuel-alvarez-alvarez) [reply tests/appsec/iast/test_iast.py:71]: Changed, thanks!

File: utils/build/docker/java/spring-boot/pom.xml (hunk 2 above)
Comment (manuel-alvarez-alvarez) [reply utils/build/docker/java/spring-boot/pom.xml:96]: Nope, not really, the same can be done with spring, removed!

**Review by robertomonteromiguel — APPROVED** [review]

**Comment by cbeauchesne** [comment]
Hi @manuel-alvarez-alvarez , reading from the conflict, I think what you tried to test isnow in the main branch. Can I close this PR ?

**Comment by manuel-alvarez-alvarez** [comment]
> Hi @manuel-alvarez-alvarez , reading from the conflict, I think what you tried to test isnow in the main branch. Can I close this PR ?

Hello @cbeauchesne the functionality this test should cover is not yet merged (at least not in the java library, [Add SQL Injection detection #3909](https://github.com/DataDog/dd-trace-java/pull/3909)), but it's taking forever :cry: 

Should and I mark the new tests as missing for all libraries so we can merge the PR?

PD: Sorry for the delay in the response, I got to improve my filtering of github notifications.

**Comment by cbeauchesne** [comment]
> Should I mark the new tests as missing for all libraries so we can merge the PR?

Yes, simply uses `@missing_feature`. The good part is that,  on your PR, you'll see it as x-passed if it's ok. Once your PR is merged on java tracer, you will do a simple PR  here to remove the decorator for java.

**Comment by manuel-alvarez-alvarez** [comment]
> > Should I mark the new tests as missing for all libraries so we can merge the PR?
> 
> Yes, simply uses `@missing_feature`. The good part is that, on your PR, you'll see it as x-passed if it's ok. Once your PR is merged on java tracer, you will do a simple PR here to remove the decorator for java.

Thanks for the input!, I've rebased from main and fixed the conflicts so it should be ready for the review and merge :smile:

**Review by cbeauchesne — APPROVED** [review]

**Review by smola — APPROVED** [review]

**Pull request merged**